src/libstd/sys/windows/fs.rs

```diff
@@ -431,6 +431,14 @@ impl FileType {
     pub fn is_dir(&self) -> bool { *self == FileType::Dir }
     pub fn is_file(&self) -> bool { *self == FileType::File }
     pub fn is_symlink(&self) -> bool { *self == FileType::Symlink }
+    #[unstable(feature = "file_types_posix", reason = "recently added API")]
+    pub fn is_block_device(&self) -> bool { false }
+    #[unstable(feature = "file_types_posix", reason = "recently added API")]
+    pub fn is_char_device(&self) -> bool { false }
```

Comment:
Sorry, my knowledge about windows is really límited, i going to investigate it and try to propose a solution. Thanks!! :smile: